### DISCUSSION TITLE
are there repos like this for other languages

### DISCUSSION THREAD

**Discussion by jamesthesnake** [kowainik/learn4haskell#371] [General]
This was a great introduction to Haskell and it kick started me to learn and use it more, do you know of any other repos like this for other languages?

--sorry for putting this in issues had no where else to go

**Comment by vrom911** [comment]
Hey, @jamesthesnake! Thanks a lot for your kind words 🤗 

Unfortunately, we are not aware of any similar to this. We came up with the idea on our own before the Hacktoberfest2020, and when trying to search if this was already existing, we could not find a thing... 
But I guess it would be nice to have a series of repos like this, even if only for Hacktoberfest format.

We will appreciate if you spread the word about this idea, and probable we could make it work with many more languages!

P.S. I moved the issue to the newly created Discussions 🙂 Feel free to use it for ideas, discussion and other questions 👌🏼
